### DISCUSSION TITLE
Item Damage Rebalancer

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#166] [Tweak Suggestions]
https://legacy.curseforge.com/minecraft/mc-mods/item-damage-rebalancer
Would be appreciated to also be able to configure set values, not only multipliers!

**Comment by Ski-z** [comment]
https://www.curseforge.com/minecraft/mc-mods/materialtweaker
and
https://www.curseforge.com/minecraft/mc-mods/material-changer
already do this.

**Reply by xJon** [reply]
Unfortunately MaterialTweaker [does not support](https://legacy.curseforge.com/minecraft/mc-mods/materialtweaker?comment=18) changes on an item-by-item basis, and Material Changer has [compatibility issues](https://legacy.curseforge.com/minecraft/mc-mods/material-changer/issues/3) with some mods' items, so neither are a good substitute.
